docs/admin/administration.md

```diff
@@ -513,6 +513,29 @@ will remain evicted until device replacement occurs).
     Full NVMe hot plug capability will be available and supported in DAOS 2.2 release.
     Use is currently intended for testing only and is not supported for production.
 
+- Unbind a newly added (hot-inserted) SSD from the kernel and bind instead to user-space
+driver so that the device can be used with DAOS:
+```bash
+$ dmg storage nvme-rebind --help
+Usage:
+  dmg [OPTIONS] storage nvme-rebind [nvme-rebind-OPTIONS]
+
+...
+
+[nvme-rebind command options]
+      -a, --pci-address= NVMe SSD PCI address to rebind.
+```
+
+To rebind a newly added NVMe SSD on a single host, run the following command (replace
+SSD PCI address and hostname with appropriate values):
+```bash
+$ dmg storage nvme-rebind -a 0000:84:00.0 -l wolf-167
+Command completed successfully
+```
```

Comment:
This is a bit verbose and potentially confusing. I'm also not sure that it's a good idea for the admin documentation to include the help output, as it adds another maintenance point.

I recommend truncating this a bit to just include the verbose description of what the command is for and the example.

Reply:
https://github.com/daos-stack/daos/pull/7913